### ISSUE TITLE
Field operating system unchanged

### ISSUE THREAD

**Issue by mvestrotech** [fusioninventory/fusioninventory-for-glpi#2596]
When the operating system elements is changed manually and an invantary is started, the previously modified information remains unchanged.

Best Regard.

**Comment by tabad** [comment]
Hello @mvestrotech,

That's because when you change a field by hand you lock it (it forbids the automatic update by FI4G). Whether you want the information will be automatically updated, don't change it by hand. Please, unlock the field you locked and force an inventory.

This is not an issue; that's the normal behavior.

Regards,

**Comment by tabad** [comment]
Hello @mvestrotech,

You're welcome. By the way, could you close this issue?

Thanks in advance. Regards,

**Comment by stonebuzz** [comment]
Hello @tabad , I have the same problem 

however I can not find an OS lock  in the tab "locks" of the PC or in the configuration of FI

How can i unlock OS field ?

GLPI 9.2.2
FI 9.2+2.0-RC1

Best regards

**Comment by tabad** [comment]
Hello @stonebuzz,

You are right, there isn't the possibility of lock/unlock any field regarding operating system in GLPI v9.2.x. I'm, sorry, I thought it was like whatever other fields.

I have read your issue #2618 and I have remembered when I upgraded from GLPI 9.1.x to GLPI 9.2.x a lot of computers was linked with more than one operating system which I think is an upgrade error. Theoretically this query shouldn't show any registry but I had a lot of them.

```sql
select items_id, operatingsystems_id from glpi_items_operatingsystems where itemtype = 'Computer' group by items_id, operatingsystems_id having count(items_id) > 1;
```

I'll study the update of operating system information further. ;)

Regards,
